packages/holocron-dev-server/__tests__/src/webpack/plugins/holocron-webpack-plugin.spec.js

```diff
@@ -55,17 +50,17 @@ describe('HolocronWebpackPlugin', () => {
             },
           })),
         },
+        version: 5,
       },
     };
-    expect(instance.apply(compiler)).toBe(undefined);
+    expect(holocronWebpackPlugin.apply(compiler)).toBe(undefined);
```

Comment:
Updated as per https://github.com/americanexpress/one-app-cli/pull/269#discussion_r622440023 to make the value more intuintive